nights_watch/abstract_store.py

```diff
@@ -42,28 +47,32 @@ def get(self, identifier):
         each of which contains a naive UTC timestamp and a possibly
         empty comment, sorted from most to least recent. Deadline
         should be a naive UTC timestamp."""
-        raise NotImplementedError()
+        raise NotImplementedError('get')
 
-    def list(self, verbose=False, paused=None, late=None):
+    @abstractmethod
+    def list(self, *, verbose=False, paused=None, late=None):
         """Return an iterator which yields dicts. If verbose is False,
         then the dicts contain only name and id, otherwise, all fields
         (same as returned by get()) are returned. If paused and/or
         late are specified, they are used to filter the results."""
-        raise NotImplementedError()
+        raise NotImplementedError('list')
 
+    @abstractmethod
     def upcoming_deadlines(self):
         """Return an iterator which yields watchers (same as returned by
         get()) that are unpaused and not yet late, sorted by deadline
         in increasing order, i.e., the watcher that will pass its
         deadline soonest is returned first."""
-        return NotImplementedError()
+        return NotImplementedError('upcoming_deadlines')
```

Comment:
This should be raise rather than return, shouldn't it? (I know, not your bug, but since I noticed it, we should fix it)